tools/check_release.py

```diff
@@ -362,11 +394,22 @@ def get_latest_library_versions(repo_path):
 
     return mbed, mbed_dev
 
+def log_results(lst, title):
+    logging.info(title)   
```

Comment:
We should do the `title` and `lst` as one statement here. Otherwise it looks weird on the output.